course_content/notebooks/numpy_intro.ipynb

```diff
@@ -995,7 +995,7 @@
      "cell_type": "code",
      "collapsed": false,
      "input": [
-      "arr = np.arange(8).reshape(2, 4)\n",
+      "arr = np.arange(2, 18, 2).reshape(2, 4)\n",
```

Comment:
Good idea! Weirdly though linspace seems to default to floats while arange defaults to (imo nicer) ints:

```python
print np.arange(2, 18, 2).reshape(2, 4)
print np.linspace(2, 16, 8).reshape(2, 4)

[[ 2  4  6  8]
 [10 12 14 16]]
[[  2.   4.   6.   8.]
 [ 10.  12.  14.  16.]]
```